### DISCUSSION TITLE
再驳开源免费论：从定价与隔离理论说起 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#19] [Announcements]
# 再驳开源免费论：从定价与隔离理论说起 - 开源之道

https://opensourceway.community/posts/open-source-economic/rethink-open-source-is-no-charge/

**Comment by ZoomQuiet** [comment]
#是也乎
其实并不是难题, 只是宣传不够以及认知偏差...

开源和免费挂勾, 可能锅在 自由软件/Free Software 和 免费软件 Freeware 在普通人看来几乎一样;
也因为, 软件开发从一开始都隐藏在硬件开发背后,
直到 Bill 创造了软件许可证, 天才的将软件代码和版权绑定在一起,
软件的工作量才慢慢被世界认可;

那么, 从以往无从付费就可以获得所有代码,
到后来付费也看不到代码的时代里,
不付费就可以拿到代码, 感觉简直太划算了,

中国人看来这是人傻代码多速来,
而在国外, 这是社区公共财产, 使用是有协议规约的...

可惜, 合约精神在中国根本从无树立, 许可证之类也纯粹给瞎子看的...
倒是很有主人翁姿态, 对自由/开源项目拿来就用, 不会就问, 却从来不知道回馈;

开源是免费的, 但是, 这只是进入免费, 一但和生产结合, 引发的一系列理解/兼容/改进/沟通/运营/....
具体实施过程中的成本, 都是具体的, 不可避免的,
也是自然/切实的, 只是很少有清晰的对比解释来让普通领导们理解:

- 采购商业软件直接使用
- 基于开源软件改进使用

两者事先/中/后 成本分布/后续变化, 值得算一笔经典的帐...

关键是:

+ 资金
+ 人力
+ 技术债务
+ 社会影响力
+ 组织潜力
+ ...


以及最后类似 CHAOSS 的开源商用项目综合收益指数, 如果能总结出来, 
就可以套在所有开源项目上了...

**Reply by lijiangsheng1** [reply]
文化的隔阂，可以引发战争，，历史的悲剧一次次的上演。
